nextflow_schema.json

```
@@ -1047,6 +1047,32 @@
             },
             "fa_icon": "fas fa-layer-group"
         },
+        "functional_decoupler": {
```

```suggestion
        "functional_decoupler_specific_options": {
```

Just for consistency with param groups above